hardhat.config.ts

```diff
@@ -96,11 +71,6 @@ const config: HardhatUserConfig = {
       1: '0x8FFA64FB50559c3Ff09a1022b84B2c5233ed8068',
     },
 
-    nftGiveawayAdmin: {
```

Comment:
Not required?

Reply:
you right, is is only used in test, I am going to remve it and update the tests

Reply:
Oh actually yes it was removed because now the role is not needed on mainnet and the test use `getUnnamedAccounts`